### PR TITLE
Adds some style to the TOC and a click handler

### PR DESCRIPTION
**Pull request by lukecampbell**

### CODE REVIEW

File: habs_portal/static/css/toc.css [hunk 1]
```diff
@@ -0,0 +1,10 @@
+.category-rollup {
+}
+
+.category-rollup a:hover {
+  text-decoration: none;
+}
+
+.panel-title.category-title > a:hover {
+  color: #3288A3;
+}
```
Comment (lukecampbell) [inline habs_portal/static/css/toc.css]: screen shot 2015-03-13 at 11 58 47 am

File: habs_portal/static/js/views/CategoryItemView.js [hunk 2]
```diff
@@ -1,13 +1,14 @@
...
- 
   add: function(subview) {
     subview.render();
     this.subviews.push(subview);
     this.$el.find('.panel-body').append(subview.el);
   },
-
+  collapse: function() {
+    this.$el.find('.collapse').collapse('toggle');
+  },
```
Comment (lukecampbell) [inline habs_portal/static/js/views/CategoryItemView.js]: Now we can click anywhere on the div and it will collapse.

**Comment by lukecampbell** [comment]
@cgalvarino

**Comment by cgalvarino** [comment]
@lukecampbell when you click on a station, it collapses or anywhere inside the stations block.  :(

**Comment by lukecampbell** [comment]
Fixed it in the latest PR

**Pull request closed without merging**